src/Test/Internal.elm

```diff
@@ -19,6 +19,14 @@ failNow record =
         (\_ _ -> [ Test.Expectation.fail record ])
 
 
+emptyDescriptionFailure : Test
+emptyDescriptionFailure =
+    failNow
+        { description = "This test has an empty description. Let's give it a nonempty one!"
```

Comment:
"Empty" typically refers to lists. Maybe, "This test has **a blank** description. Let's give it a **useful** one!"